src/context/__tests__/Auth.tsx

```diff
@@ -0,0 +1,86 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import { sub } from 'date-fns';
+import add from 'date-fns/add';
+import formatISO from 'date-fns/formatISO';
+import { type ReactElement } from 'react';
+import { sessionStorageKey, userStorageKey } from 'services/auth';
+import { AuthProvider, useAuth } from '../Auth';
+import AppContent from '../Content';
+
+const renderAuth = (ui: ReactElement) => {
+    return render(<AppContent>{ui}</AppContent>);
+};
+const fakeUser = 'FooBar123';
+const userMissing = 'No User Found';
+
+const FakeComponent = () => {
+    const { user } = useAuth();
+
+    return user ? user : userMissing;
+};
```

Comment:
Super basic setup to render out a fake component just to test the user value from `useAuth`. Obviously we could handle this with a real component test elsewhere but I like having this little stand alone test since there is logic inside the context provider.